File: Back End/notifications/src/main/java/com/theelite/notifications/model/Notification.java
package com.theelite.notifications.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;
import java.util.UUID;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Notification {
    private String notification;
    private Date date;
    private UUID doorId;
    private UUID accountId;

    public Notification(String notification, String doorId) {
        this.date = new Date();
        this.notification = notification;
        this.doorId = UUID.fromString(doorId);
    }
}

File: Front End/Android/app/src/main/java/com/theelite/portal/Objects/Notification.kt
package com.theelite.portal.Objects

import io.realm.RealmObject
import java.util.*

open class Notification() : RealmObject() {
    lateinit var notification: String
    lateinit var date: Date
    lateinit var accountId: String
    lateinit var doorId: String

    constructor(notification: String, date: Date, accountId: String, doorId: String) : this()
}
